NetKVM/Common/ParaNdis-TX.h

```diff
@@ -255,6 +294,18 @@ class CParaNdisTX : public CParaNdisTemplatePath<CTXVirtQueue>, public CNdisAllo
 
     void CompleteOutstandingNBLChain(PNET_BUFFER_LIST NBL, ULONG Flags = 0);
     void CompleteOutstandingInternalNBL(PNET_BUFFER_LIST NBL, BOOLEAN UnregisterOutstanding = TRUE);
+
+#if MAX_FRAGMENTS_IN_ONE_NB
+    bool AllocateExtraPages();
+    void FreeExtraPages();
```

Comment:
Why these are public: Allocate and Free? They are used only by CParandisRx itself.

Reply:
The mistake I made carelessly.